pkg/server/server_test.go

```diff
@@ -37,7 +37,7 @@ func (s *TestServerSuite) TestServer() {
 
 	fake.MockKeycloakCertsCall(s.T())
 	// Setting up the routes.
-	err := srv.SetupRoutes()
+	err := srv.SetupRoutes("8091")
```

Comment:
```suggestion
	err := srv.SetupRoutes("8091") // uses a different proxy port than the default one to avoid collision with other concurrent tests
```

Reply:
thanks for your suggestion, I've applied the change you suggested and renamed the constant to `proxy.DefaultPort`